www/assets/locales/locale-en.json

```diff
@@ -219,7 +219,10 @@
     "needs-restart": "Restart app to apply changes.",
     "appearance": {
       "title": "Appearance",
-      "dark": "Dark Mode",
+      "app-mode": "App Mode",
```

Comment:
I was just about to translate this, but found this title a bit peculiar. It's vague enough that I had to come here to double check what it was. Wouldn't "Theme" be more to the point?

Reply:
The theme is the colours the UI will use, but each theme has two variations/modes, dark and light.

Reply:
Yeah I admit App Mode is not an ideal name. But I thought the three available options `Use System Theme`, `Light`, and `Dark` should make it clear what it is. Do you have an idea for a better name?

Reply:
Maybe theme mode would be better.

Reply:
Yes, Theme Mode sounds good. Go ahead and change it if you want. I don't want to make a PR just for that ;)